### PR TITLE
Remove special case rendering of thumbnail in show

### LINKED ISSUE #785: Get signature image the same way we get other images
Andrew wanted to add alt text in the mark down, but it wasn't showing up, because we're not just rendering the markdown for that. Revisit this to make it less surprising.

### PR DESCRIPTION
**Pull request by mccalluc**
... at the cost of more model logic. Tweak MD to match. 
fix #785. @afred, please review.

### CODE REVIEW

**Comment by mccalluc** [comment]
This is going to conflict with all the MD changes: Best probably to pull out the small code changes and re-PR when things have settled down.

**Comment by mccalluc** [comment]
(Also: Signature images will start showing up in the blurbs above search results: not necessarily a bad thing.)

File: app/views/exhibits/civil-rights.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 # Civil Rights
 
-!["The Blue Marble" Source: NASA](https://s3.amazonaws.com/americanarchive.org/exhibits/ClimateChange_SignatureImage.jpg "The Blue Marble" Source: NASA")
-
 ## Summary
 
+!["The Blue Marble" Source: NASA](https://s3.amazonaws.com/americanarchive.org/exhibits/ClimateChange_SignatureImage.jpg "The Blue Marble" Source: NASA")
 TODO
```
Comment (afred) [inline app/views/exhibits/civil-rights.md]: @caseyedavis12 @mccalluc Mismatched double quotes after img source. Also, what does "TODO" do?

File: app/views/exhibits/civil-rights/rename-me.md [hunk 2]
```diff
@@ -1,9 +1,8 @@
 # Civil Rights
```
Comment (afred) [inline app/views/exhibits/civil-rights/rename-me.md]: @caseyedavis12 I'm guessing we don't want to keep `rename-me`?

File: app/views/exhibits/civil-rights.md (hunk 1 above)
Comment (afred) [inline app/views/exhibits/civil-rights.md]: @caseyedavis12 also.. are we sure we want the "Blue Marble" image for the "Civil Rights" collection?

File: app/views/exhibits/climate-change.md [hunk 3]
```diff
@@ -1,9 +1,8 @@
 # Climate Change Conversations: Causes, Impacts, Solutions
 
-!["The Blue Marble" Source: NASA](https://s3.amazonaws.com/americanarchive.org/exhibits/ClimateChange_SignatureImage.jpg "The Blue Marble" Source: NASA")
-
 ## Summary
 
+!["The Blue Marble" Source: NASA](https://s3.amazonaws.com/americanarchive.org/exhibits/ClimateChange_SignatureImage.jpg "The Blue Marble" Source: NASA")
```
Comment (afred) [inline app/views/exhibits/climate-change.md]: More mismatched quotes.. same as in `civil-rights.md`

File: app/views/exhibits/climate-change/solutions.md [hunk 4]
```diff
@@ -11,6 +9,7 @@
 
 ## Summary
 
+![Wind Turbines (CC-BY-SA 3.0 Image by Fred J, userpage: commons.wikimedia.org/wiki/User:Fred_J](https://s3.amazonaws.com/americanarchive.org/exhibits/ClimateChange_Section5_Solutions.jpg "Wind Turbines (CC-BY-SA 3.0 Image by Fred J, userpage: commons.wikimedia.org/wiki/User:Fred_J)")
```
Comment (afred) [inline app/views/exhibits/climate-change/solutions.md]: @mccalluc @caseyedavis12 double check that using parentheses within the Alt-text works

File: app/views/exhibits/station-histories.md [hunk 5]
```diff
@@ -1,9 +1,8 @@
 # Documenting and Celebrating Public Broadcasting Station Histories
 
-![Early WGBH Control Room](https://s3.amazonaws.com/americanarchive.org/exhibits/AAPB_Exhibit_StationHistories_image0.jpg)
-
 ## Summary
 
+![Early WGBH Control Room](https://s3.amazonaws.com/americanarchive.org/exhibits/AAPB_Exhibit_StationHistories_image0.jpg)
```
Comment (afred) [inline app/views/exhibits/station-histories.md]: Missing the title text here.

**Comment by afred** [comment]
@mccalluc @caseyedavis12 I'll make these fixes, rebase, and re-push. Stand by.

**Comment by afred** [comment]
@mccalluc pushed changes to md. Sanity check please?

**Comment by afred** [comment]
and just added titles to @sroosa exhibit pages

**Pull request merged**